### PR TITLE
feat: Add temp ga integration to webloader - RKT-8767

### PR DESCRIPTION
**Pull request by timcubb**
RKT-8767

### CODE REVIEW

File: src/webloader.js [hunk 1]
```diff
@@ -142,6 +143,8 @@ function main() {
 
 	// If uid is empty, or evolvLazyUid is set - don't run the webloader until a uid is set using evolv.setUid().
```
Comment (tfoster) [inline src/webloader.js:144]: The gaIntegration should only execute if the flag is set. Not if it UID is not set.

File: src/integrations/ga.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+let pollAttempt = 0;
+const maxPollAttempts = 60;
+
+function gaIntegration() {
+    if (pollAttempt > maxPollAttempts) {
+        console.log('Evolv: unable to set uid');
+        return;
+    }
+
+    if (typeof ga === 'function' ) {
```
Comment (tfoster) [inline src/integrations/ga.js:10]: ga should be window.ga

**Review by tfoster — CHANGES_REQUESTED** [review]

File: src/webloader.js (hunk 1 above)
Comment (timcubb) [reply src/webloader.js]: fixed

File: src/integrations/ga.js (hunk 2 above)
Comment (timcubb) [reply src/integrations/ga.js]: fixed

**Comment by timcubb** [comment]
@tfoster - updated with changes to test cases

**Review by tfoster — APPROVED** [review]

**Comment by timcubb** [comment]
hotfix-v1.8

**Pull request merged**